streams/src/main/java/org/apache/kafka/streams/processor/internals/SourceNodeRecordDeserializer.java

```diff
@@ -35,6 +35,7 @@
         this.deserializationExceptionHandler = deserializationExceptionHandler;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Yes. Also a side cleanup to suppress warning about deprecated call to `rawRecord.checksum()`